server/container/guice/cassandra-guice/src/main/java/org/apache/james/CassandraJamesServerMain.java

```diff
@@ -98,10 +98,15 @@
         new TikaMailboxModule(),
         new SpamAssassinListenerModule());
 
+    public static Module CASSANDRA_MODULE_AGGREGATE = Modules.combine(
```

Comment:
Module name is not ok, it fails to describe what it aggregates

Reply:
Could you propose a better name?

Reply:
BTW we have similar terms in other places: `IN_MEMORY_SERVER_AGGREGATE_MODULE` for memory.